### DISCUSSION TITLE
Is it possible to use ScheduleDate and RepeatInterval at the same time in the Shiny.Notifications?

### DISCUSSION THREAD

**Discussion by sagarchopranimbus** [shinyorg/shiny#1429] [Q&A]
I had a requirement where I wanted to schedule a notification for a specific date or time and then repeat it continuously (every min or every hour) until user completes the task told to do  .

For example:
I want to start showing notification from tomorrow 11 AM. So I should be able to schedule it for 11 am tomorrow. and once the first notification is triggered it should repeat after every 10 minutes to remind the user that he has task to perform. the notification will only stop once user finishes the task.

However, As per Shiny docs we cannot use ScheduleDate and RepeatInterval at the same time.

Is it possible to do it some other way?

**Comment by munkii** [comment]
Before we moved to push notifications generated in our back end we also had a requirement to remind a users to do something at the same time every day. 

Using Shiny Local notifications we would initially schedule 10 days worth of reminders. 

Then on app startup we would run that same code to ensure there was always 10 days worth of reminders. Obviously if the user doesn’t open your app for 11 days or more the reminders will stop but if you have been reminding them for 10 days and they still haven’t opened the app its probably time to stop hassling them.
